libs/dyn/dynloc/locations_test.go

```diff
@@ -0,0 +1,100 @@
+package dynloc
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/databricks/cli/libs/dyn"
+	assert "github.com/databricks/cli/libs/dyn/dynassert"
+	"github.com/databricks/cli/libs/dyn/merge"
+	"github.com/databricks/cli/libs/dyn/yamlloader"
+	"github.com/stretchr/testify/require"
+)
+
+func load(t *testing.T, path string) dyn.Value {
+	matches, err := filepath.Glob(path + "/*.yml")
+	require.NoError(t, err)
+	require.NotEmpty(t, matches)
+
+	// Load all files.
+	vout := dyn.NilValue
+	for _, match := range matches {
+		buf, err := os.ReadFile(match)
+		require.NoError(t, err)
+
+		v, err := yamlloader.LoadYAML(match, bytes.NewBuffer(buf))
+		require.NoError(t, err)
+
+		vout, err = merge.Merge(vout, v)
+		require.NoError(t, err)
+	}
```

Comment:
I'd keep unit test as simple input -> output table. If you add parsing and merging logic, I'd move that to acceptance tests, so that it actually tests real behavior and not artificial.